### DISCUSSION TITLE
Is there a way to add third party search engine?

### DISCUSSION THREAD

**Discussion by Raj2032** [bromite/bromite#1789] [Q&A]
Hey guys I wanted to add https://www.startpage.com/ as my default search engine on bromite but it doesn't seem to have the option to. Is there a way I can add this as my default search engine?

![image](https://user-images.githubusercontent.com/29293830/153738116-916c986f-560a-43b5-93c1-38827374c560.png)

I have done the first 2 steps, but then it says `The search engine can now be selected as a default.`

What does that exactly mean. How do I do that?

**Comment by csagan5** [comment]
The [wiki page](https://github.com/bromite/bromite/wiki/SearchEngines) has another paragraph above that:
> You can add any search engine which implements [OpenSearch](https://developer.mozilla.org/en-US/docs/Web/OpenSearch) to the list by following these steps:

You should first check that the search engine implements OpenSearch, otherwise it will not work.
